windows/CommonSettings.props.example

```diff
@@ -11,6 +11,10 @@
          set to the root of your Python installation. If your Python installation
          does not contain debug libraries, debug build will not work. -->
         <PythonSupport>false</PythonSupport>
+        <!-- NOTE: If Matlab support is enabled, MatlabDir (below) needs to be
+         set to the root of your Matlab installation, and Python support has to
+         be disabled. Only works in Release mode. -->
```

Comment:
@pavlejosipovic @lunzueta I'm also working on matcaffe. I can fix this and send PR to lunzueta's repo.